app/src/main/res/values/styles.xml

```diff
@@ -425,11 +429,14 @@
   </style>
   <!-- Continue Lesson Button -->
   <style name="ContinueLessonButton" parent="TextAppearance.AppCompat.Widget.Button">
-    <item name="android:layout_width">144dp</item>
+    <item name="android:layout_width">wrap_content</item>
```

Comment:
Why is minWidth set to 48dp below when elsewhere we set the minWidth to the previously fixed width?

Also, similar question to an earlier one: will the button actually look correct if the text is larger?

Ditto on both for StartOverLessonButton.

Reply:
Thanks--I think whether this approach works very much depends on the parent layout (the fact that a ConstraintLayout is being used ensures that the buttons always look correct, but other parent layouts may introduce issues with this approach I think).